manager/integration/tests/conftest.py

```diff
@@ -173,3 +179,42 @@ def pytest_collection_modifyitems(config, items):
         for item in items:
             if "upgrade" in item.keywords:
                 item.add_marker(skip_upgrade)
+
+
+def pytest_exception_interact(call, report):
+
+    # Get case name
+    case_name = str(report).split()[1]
+    case_name = case_name.split('/')[1].replace('\'', '')
+
+    # Use API gen support bundle
+    client = get_longhorn_api_client()
+    url = client._url.replace('schemas', 'supportbundles')
+    data = {'description': case_name, 'issueURL': case_name}
+    res = requests.post(url, json=data).json()
+
+    id = res['id']
+    name = res['name']
+
+    # Wait support bundle complete
+    support_bundle_url = '{}/{}/{}'.format(url, id, name)
+    for i in range(RETRY_EXEC_COUNTS):
+        res = requests.get(support_bundle_url).json()
+
+        if res['progressPercentage'] == 100:
+            break
+        else:
+            time.sleep(RETRY_INTERVAL_LONG)
+
+    # Download support bindle
+    download_url = '{}/download'.format(support_bundle_url)
+
+    r = requests.get(download_url, allow_redirects=True)
+
+    os.makedirs("support_bundle", exist_ok=True)
+
+    file_cnt = len(os.listdir("support_bundle"))
+
+    if file_cnt < MAX_SUPPORT_BINDLE_NUMBER:
```

Comment:
If >= max, please leave a message about ignoring the bundle download because of avoiding overwhelming the disk usage.

Reply:
Modified, please see https://github.com/longhorn/longhorn-tests/pull/879#issuecomment-1040301027